<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admcontrol</class>
 <widget class="QDialog" name="admcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理员操作</string>
  </property>
  <widget class="QTabWidget" name="addbook">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>18</y>
     <width>601</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>添加书籍</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(118, 118, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px ;
color: rgb(255, 255, 255);
background-color: rgb(26, 188, 156);</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px ;</string>
     </property>
     <property name="text">
      <string>请选择类别：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px ;
background-color: rgb(199, 199, 199);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="isbnEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px ;
color: rgb(139, 139, 139);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>ISBN：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="publisher">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px ;
color: rgb(139, 139, 139);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>出版社：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="author">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px ;
color: rgb(139, 139, 139);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>作者：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px ;
gridline-color: rgb(0, 85, 255);
color: rgb(139, 139, 139);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>书名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(118, 118, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(118, 118, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(118, 118, 118);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>381</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(222, 222, 222);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>comboBox</zorder>
    <zorder>resultLabel</zorder>
    <zorder>isbnEdit</zorder>
    <zorder>publisher</zorder>
    <zorder>author</zorder>
    <zorder>name_1</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>书籍数据</string>
    </attribute>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	alternate-background-color: rgb(255, 255, 255);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>请选择搜索目标：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>621</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(222, 222, 222);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="searchBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>14</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(232, 232, 232);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>读者状态控制</string>
    </attribute>
    <widget class="QLineEdit" name="readername_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>解禁读者名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readername">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>封禁读者名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>237</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>读者数据</string>
    </attribute>
    <widget class="QLineEdit" name="name_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,0);
	border:none;
	border-bottom:1px solid black;}</string>
     </property>
     <property name="placeholderText">
      <string>请选择搜索目标：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>name_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>author</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>publisher</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>isbnEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
